src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -191,6 +191,55 @@ public void FailFast_ExpectFailureExitCode()
             }
         }
 
+        [Trait(XunitConstants.Category, XunitConstants.IgnoreForCI)] // fail fast crashes the process
+        [Fact]
+        public void FailFast_InnerExceptionStackTrace()
+        {
+            var psi = new ProcessStartInfo();
+            psi.RedirectStandardError = true;
+
+            using (RemoteInvokeHandle handle = RemoteInvoke(
+                () => { Environment.FailFast("message", new ArgumentException("bad arg")); return SuccessExitCode; },
+                new RemoteInvokeOptions { StartInfo = psi }))
+            {
+                Process p = handle.Process;
+                handle.Process = null;
+                p.WaitForExit();
+                String consoleOutput = p.StandardError.ReadToEnd();
+                Assert.True(consoleOutput.Contains("Exception details:"));
```

Comment:
All these can change to just eg
`Assert.Contains("Exception details:", consoleOutput);`